cczoo/psi/gramine/anolisos-psi-gramine-sgx-dev.dockerfile

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BASE_IMAGE=gramine-sgx-dev:v1.2-anolisos
```

Comment:
Need to update this dockerfile with the same comments for the gramine-sgx-dev:v1.2-anolisos.

Reply:
Thanks to Zongmin's advice, I will modify it accordingly